main.py

```diff
@@ -138,38 +141,21 @@ def download_hashtags_last_7_days(hashtag_labels):
         download_video(hashtag_label, 50, post_from_last_7_days)
     os.system('python sort_best.py')
 
-# download all hashtags daily
+# tasks to be done daily
 def cronjob():
     schedule.every(2).hours.do(download_hashtags_last_7_days, hashtag_labels)
-    schedule.every().day.at("00:00").do(remove_files)
+    schedule.every().day.at("00:00").do(lambda: zip_non_used_files())
```

Comment:
Why do we need the lambda expression here? Shouldn't we just call the `zip_non_used_files` instead?

 ```suggestion
    schedule.every().day.at("00:00").do(zip_non_used_files)
```

Reply:
Unfortunetelly it doesn't work with just the regular function calling, because this specific function does not receives any arguments. Calling it without lambda raises an error, but does not kill the process. I used this as reference:
https://stackoverflow.com/questions/40781062/pass-parameters-to-schedule

Reply:
I still didn't understand well how the stack overflow link relates to our case since we are not passing any arguments to `zip_non_used_files`, but I also don't see any problem with the current solution, we can keep it as it is.